### DISCUSSION TITLE
您好，关于WS接口的格式探讨一下。

### DISCUSSION THREAD

**Discussion by Minstrel-Pal** [oliverkirk-sudo/WarpGPT#21] [Ideas]
/r/v1/chat/completions/ws这个接口的格式能否改成/r/ws/v1/chat/completions，因为不少客户端的接口末尾都是写死的，都是以v1/chat/completions结尾，所以我想是不是能改一下格式。

**Comment by oliverkirk-sudo** [comment]
可以改
